src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -125,4 +127,13 @@ private void handlePersonPanelSelectionChangedEvent(PersonPanelSelectionChangedE
         mainWindow.loadPersonPage(event.getNewSelection());
     }
 
+    //@@author A0135998H
+    @Subscribe
+    private void handleViewListChangedEvent(ViewListChangedEvent event) {
+        logger.info(LogsCenter.getEventHandlingLogMessage(event));
+        int firstSpaceIndex = event.toString().indexOf(" ");
+        String typeOfList = event.toString().substring(0, firstSpaceIndex);
```

Comment:
Erm wow I don't think this is the way to write code? You don't parse an event and turn it into a listType? For example some of the events uses `event.data` not toString?